nifi-frontend/src/main/frontend/apps/nifi/src/app/pages/parameter-contexts/ui/parameter-context-listing/parameter-context-table/parameter-context-table.component.ts

```diff
@@ -86,6 +86,19 @@ export class ParameterContextTable {
         return this.canRead(entity) && entity.component ? entity.component.description : '';
     }
 
+    formatBoundedProcessGroups(entity: ParameterContextEntity): string {
+        return this.canRead(entity) && entity.component ? this.getHighLevelParameterContexts(entity.component) : '';
+    }
+
+    private getHighLevelParameterContexts(component: ParameterContext) {
+        const componentIds: string[] = component.boundProcessGroups.map(group => group.component.id)
+        const boundedProcessGroupNames: string[] =  component.boundProcessGroups
+            .filter(group => !componentIds.includes(group.component.parentGroupId))
+            .map(group => group.component.name);
+
+        return boundedProcessGroupNames.length <= 1 ? boundedProcessGroupNames.toString() : '(shared)'
```

Comment:
Instead of returning `(shared)` for multiple groups, I recommend returning the number of process groups.

Reply:
That will be nice! I adjusted the code.